run.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+#temporeraly import variables
+export $(cat .env)
+#run postgres and hasura in containers, give 2 seconds to start postgres
+docker-compose up -d
+sleep 2
+#init database with basic tables
+docker exec -ti juno_postgres psql -f /root/schema/validator.sql -d $POSTGRES_DB_NAME -U $POSTGRES_USER_NAME
+docker exec -ti juno_postgres psql -f /root/schema/pre_commit.sql -d $POSTGRES_DB_NAME -U $POSTGRES_USER_NAME
+docker exec -ti juno_postgres psql -f /root/schema/block.sql -d $POSTGRES_DB_NAME -U $POSTGRES_USER_NAME
+docker exec -ti juno_postgres psql -f /root/schema/transaction.sql -d $POSTGRES_DB_NAME -U $POSTGRES_USER_NAME
+#create config.toml, put values from .env file to config.toml
+cat sample.config.toml >> config.toml
```

Comment:
So the flow here is:

- Populate `.env`
- Copy sample.config.toml to config.toml
- Fill config.toml from env
- Remove config.toml (Docker container will now have it)

This seems a bit off to me...Can we not just have a `docker.config.toml` the user populates and this file is simply passed/copied to the container. Seems we can skip a lot of steps.

Reply:
one problem i'm not sure how to solve: same variables required during different steps of process. For example docker-compose.yaml require postgres vars to be set, than Juno need them few steps later, and docker-compose can consume only .env files, and basically thats why i created all those steps.

Reply:
I see. That makes sense. Can we at least add some newlines and spacing to comments to make this easier to read:

e.g.
```
docker exec -ti juno_postgres psql -f /root/schema/transaction.sql -d $POSTGRES_DB_NAME -U $POSTGRES_USER_NAME

# create config.toml, put values from .env file to config.toml
cat sample.config.toml >> config.toml
```